app/bundles/CoreBundle/Event/EntityExportEvent.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
```

Comment:
Please declare strict types on all new classes.
```suggestion

declare(strict_types=1);

```

Reply:
fixed